pkg/graph/schema.graphql

```diff
@@ -46,6 +46,7 @@ type ModelPlan {
   itTools: PlanITTools!
   status: ModelStatus!
   isFavorite: Boolean!
+  crTdls: [PlanCrTdl!]
```

Comment:
If you want to go buck wild (TDD, baby!) I could see value in writing a test in `plan_cr_tdl_test.go` that tests `PlanCrTdlsGetByModelPlanID` with no results to see if we get back `null` or if we get `[]`!

Reply:
Ok cool, that makes sense. I _think_ this still returns an empty array but after reading the [schema documentation](https://graphql.org/learn/schema/#object-types-and-fields) it does make sense to add the bang after the array too since we want to ensure we get an array back -  
> [Episode!]! represents an array of Episode objects. Since it is also non-nullable, you can always expect an array (with zero or more items) when you query the appearsIn field. And since Episode! is also non-nullable, you can always expect every item of the array to be an Episode object.

I'll change this and add some tests!